File: RestauranteWebApi/RestauranteService/Service/Model/PedidoModel/ListarModel.cs
using RestauranteDominio;
using RestauranteDominio.Enum;

namespace RestauranteService.Service.PedidoModel
{
   public class ListarModel
    {
        public int PedidoId { get; set; }
        public int ProdutoId { get; set; }
        public string ProdutoNome { get; set; }
        public double PedidoValor { get; set; }
        public int QuantidadeProduto { get; set; }
        public StatusPedidoEnum StatusEnum { get; set; }
    }
}

File: RestauranteFront/restaurante-app/src/app/home/pedido/models/listarModel.ts
import { StatusPedidoEnum } from "src/app/const/status-pedido.enum";

export interface ListarModel { 

    pedidoId: number;
    produtoId: number;
    produtoNome: string;
    pedidoValor: number;
    quantidadeProduto: number;
    statusEnum: StatusPedidoEnum;
}